netaddr.go

```diff
@@ -96,13 +90,7 @@ const (
 	v6Loopback = "\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x01"
```

Comment:
This could now be `v6Loopback = [16]byte{15: 1}`, I think.

(Can't do mapped4Prefix, until golang.org/issue/395 gets fixed, sadly.)